src/main/java/uk/gov/justice/hmpps/prison/api/model/Alert.java

```diff
@@ -81,6 +82,10 @@ public class Alert {
     @JsonProperty("dateExpires")
     private LocalDate dateExpires;
 
+    @Schema(description = "Datetime the alert was last modified", example = "2020-08-20T12:34:56")
+    @JsonProperty("dateModified")
+    private LocalDateTime dateModified;
```

Comment:
should really be called `dateTimeModified` since it doesn't just hold the date.  I'm not a great fan of prefixing the name with the type either, but that seems to be the way the others are done in this class.  Should really be `modifiedDateTime`.  Also would be good to include more information about the format in the `description` - see [CourtHearingDateAmendment](https://github.com/ministryofjustice/prison-api/blob/253475ab2362a7bc6452ca25391dc80931e21560/src/main/java/uk/gov/justice/hmpps/prison/api/model/CourtHearingDateAmendment.java#L22) for an example